openpype/hosts/nuke/startup/custom_write_node.py

```diff
@@ -1,6 +1,11 @@
+""" OpenPype custom script for setting up write nodes for non-publish """
 import os
 import nuke
-from openpype.hosts.nuke.api.lib import set_node_knobs_from_settings
+import nukescripts
+from openpype.hosts.nuke.api.lib import (
+    set_node_knobs_from_settings,
+    get_nuke_imageio_settings
+)
 
 
 frame_padding = 5
```

Comment:
How about we take the frame padding value from Anatomy templates rather than to hardcode it?